### PR TITLE
[0.3] Backpack/CRUD 3.3 support

### PR DESCRIPTION
**Pull request by jguyomard**
This PR adds Backpack\CRUD v3.3.* supports (and drop Laravel 5.4 support).

### CODE REVIEW

File: src/Http/Controllers/Admin/Menu/ItemController.php
Comment (ghost) [inline src/Http/Controllers/Admin/Menu/ItemController.php]: Why don't you use the $menuId variable defined above?

**Review by ghost — COMMENTED** [review]
I installed this version along with Laravel and Backpack. I did not find any issues defining and rendering a custom menu. The tests finish successfully. So It seems good to me.

I just posted a question, I wait for your response before approval.

File: src/Http/Controllers/Admin/Menu/ItemController.php
Comment (jguyomard) [reply src/Http/Controllers/Admin/Menu/ItemController.php]: Fixed

**Review by ghost — APPROVED** [review]

**Review by shaoshiva — APPROVED** [review]

**Pull request merged**
